optuna/integration/botorch.py

```diff
@@ -542,10 +578,13 @@ def sample_relative(
         if len(search_space) == 0:
             return {}
 
-        trials = study.get_trials(deepcopy=False, states=(TrialState.COMPLETE,))
+        completed_trials = study.get_trials(deepcopy=False, states=(TrialState.COMPLETE,))
+        running_trials = study.get_trials(deepcopy=False, states=(TrialState.RUNNING,))
+        trials = completed_trials + running_trials
 
         n_trials = len(trials)
-        if n_trials < self._n_startup_trials:
+        n_completed_trials = len(completed_trials)
+        if n_completed_trials < self._n_startup_trials:
```

Comment:
Do you think it's more consistent to use `n_trials < self._n_startup_trials` when `consider_running_trials=True`?

Reply:
@HideakiImamura I'd also like your opinion.

Reply:
I agree with this idea.

Reply:
Fixed as suggested.